roles/1-prep/tasks/hardware.yml

```diff
@@ -1,17 +1,38 @@
+- name: Install package 'cloud-guest-utils' so growpart is available for expanding rootfs partition during boot
+  package:
+    name: cloud-guest-utils    # 2022-03-15: For RasPiOS especially.  Ubuntu has still pre-installed, for use with cloud-init.
+    state: present
+
```

Comment:
Good point.  Installing `cloud-guest-utils` is indeed currently overkill on RasPiOS.

`/usr/bin/growpart` is only 22 kB however, and the total package size is also tiny:

```
root@box:~# apt show cloud-guest-utils | grep Size

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Installed-Size: 62.5 kB
Download-Size: 23.7 kB
```
```
root@box:~# dpkg-query -L cloud-guest-utils
/.
/usr
/usr/bin
/usr/bin/ec2metadata
/usr/bin/growpart
/usr/bin/vcs-run
/usr/share
/usr/share/doc
/usr/share/doc/cloud-guest-utils
/usr/share/doc/cloud-guest-utils/NEWS.Debian.gz
/usr/share/doc/cloud-guest-utils/changelog.Debian.gz
/usr/share/doc/cloud-guest-utils/changelog.gz
/usr/share/doc/cloud-guest-utils/copyright
/usr/share/man
/usr/share/man/man1
/usr/share/man/man1/growpart.1.gz
```

Currently I'm inclined to continue installing it for all OS's, in case such infra utilities later help others.

But if others feel otherwise (e.g. if growpart proves to be unmaintained and/or causing problems?) then we could later try to remove it completely?